### PR TITLE
new recipe for booster v0.1.2

### PR DESCRIPTION
**Pull request by BenjaminDoran**

### CODE REVIEW

File: B/booster/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Collection of sources required to build SFML
+sources = [
+    GitSource("https://github.com/evolbioinfo/booster.git", "a5f5044b192db7ab5572f44ef46abaea007f5ce9")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/booster
+install_license ${WORKSPACE}/srcdir/booster/LICENSE
+
```
Comment (giordano) [inline B/booster/build_tarballs.jl:17]: The license should be installed automatically, if you want you can remove all of this
```suggestion
```

File: B/booster/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+# compile booster
+cd ${WORKSPACE}/srcdir/booster/src
+make -j${nproc}
```
Comment (giordano) [inline B/booster/build_tarballs.jl:20]: I don't really like their default choices.  We want to use clang on BSD systems:
```suggestion
make -j${nproc} CC=${CC} CFLAGS_OMP="-Wall -g -fopenmp"
```

File: B/booster/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
+    ExecutableProduct("booster", :booster),
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = Dependency[
+    Dependency("OpenMPI_jll", v"4.1.2"),
```
Comment (giordano) [inline B/booster/build_tarballs.jl:37]: Is MPI used at all?  Doesn't look like it?

File: B/booster/build_tarballs.jl [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
+    ExecutableProduct("booster", :booster),
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = Dependency[
```
Comment (giordano) [inline B/booster/build_tarballs.jl:36]:
```suggestion
dependencies = [
    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"); platforms=filter(!Sys.isbsd, platforms)),
    Dependency(PackageSpec(name="LLVMOpenMP_jll", uuid="1d63c593-3942-5779-bab2-d838dc0a180e"); platforms=filter(Sys.isbsd, platforms)),
```

File: B/booster/build_tarballs.jl (hunk 3 above)
Comment (BenjaminDoran) [reply B/booster/build_tarballs.jl:34]: The binary needs the potential to be used with MPI, i.e. the binary has an argument for number of cpu's to use, and needs to be compiled with the MPI flags available.

if there is a best practice to get this effect I am open to it.

Comment (giordano) [reply B/booster/build_tarballs.jl]: I don't see any reference at all to MPI in the code. Are you sure you aren't confused with OpenMP, which, as confusing as it is, it's something completely different?

**Comment by BenjaminDoran** [comment]
Yeah (super confusing), I think you are right. I added a new commit that
removed this dependency. Testing locally it compiled and worked correctly.
It is the latest commit in the pull request which also passed.

-Benjamin

**Pull request merged**